std/bigint.d

```diff
@@ -145,6 +145,38 @@ public:
         assertThrown!ConvException(BigInt(r4));
     }
 
+    /**
+     * Construct a `BigInt` from a sign and a magnitude. The magnitude is an
```

Comment:
Add to newlines after the first sentence or else the whole paragraph is interpreted as a big title by ddoc.